draft-ietf-quic-recovery.md

```diff
@@ -767,7 +768,7 @@ initial RTT estimate.
 
 ## Discarding Keys and Packet State {#discarding-packets}
 
-When packet protection keys are discarded (see Section 4.9 of {{QUIC-TLS}}),
+When packet protection keys are discarded from an entire encryption level (see Section 4.9 of {{QUIC-TLS}}),
```

Comment:
I believe that we refer to the 1-RTT key update procedure as "discarding" the old generation of keys.  My understanding is that the loss of recovery state described here does not apply to 1-RTT key update.

Reply:
```suggestion
When Initial and Handshake packet protection keys are discarded (see Section 4.9 of {{QUIC-TLS}}),
```